features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Configuration/RuntimeConfiguration.jsx

```diff
@@ -482,7 +488,9 @@ export default function RuntimeConfiguration() {
                         >
                             <Grid item xs={12} style={{ marginBottom: 30, position: 'relative' }}>
                                 <Paper className={classes.paper} elevation={0}>
-                                    <APISecurity api={apiConfig} configDispatcher={configDispatcher} />
+                                    <RuntimeConfigurationContext.Provider value={{ setDisabled }}>
```

Comment:
I Don't think we need to introduce a new context here, The `configDispatcher` is already there in the `AuthorizationHeader.jsx` you can define new action for that dispatcher if you need or a better way is to do validate in an `useEffect` when the authorization header(`authorizationHeader` property in the reducer state) get changed

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/10192/commits/82a33e7c752652ee92ae647ace14fe9c218e6650